lib/mcb/commands/providers/rollover.rb

```diff
@@ -0,0 +1,18 @@
+summary "Create a copy of provider's courses for the next recruitment cycle"
+
```

Comment:
It'd be helpful to hint at the optional provider codes by defining the usage string. I had to read the code to work it out.